packages/experimental/package.json

```
@@ -0,0 +1,31 @@
+{
+	"name": "@woocommerce/experimental",
+	"version": "1.0.0",
+	"description": "WooCommerce experimental components.",
+	"author": "Automattic",
+	"license": "GPL-3.0-or-later",
+	"keywords": [
+		"wordpress",
+		"woocommerce",
+		"experimental"
+	],
+	"homepage": "https://github.com/woocommerce/woocommerce-admin/tree/main/packages/experimental/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/woocommerce/woocommerce-admin.git"
+	},
+	"bugs": {
+		"url": "https://github.com/woocommerce/woocommerce-admin/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.11.2",
+		"@wordpress/components": "10.0.0"
```

I thought this might be your tests failing issue, but no luck
```suggestion
		"@wordpress/components": "11.0.0"
```

---

Thanks for giving it a shot.  Think I've got it.  It was mocking the index file in tests that caused the issue.  Fixed up in https://github.com/woocommerce/woocommerce-admin/pull/5961/commits/7ca27ec689fa313f087d5f54596f38a55386cdcc